package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>cmake_common_scripts</name>
+  <version>0.1.0</version>
+  <description>Contains common cmake script, macros and utilst</description>
+  <maintainer email="Levi.Armstrong@swri.org">Levi Armstrong</maintainer>
+  <license>Apache 2.0</license>
+  <author>Levi Armstrong</author>
+
+  <test_depend>gtest</test_depend>
+  <test_depend>lcov</test_depend>
+  <test_depend>libclang-dev</test_depend>
```

Comment:
Need iwyu, clang-format?

Reply:
I added iwyu, but currently this does not use clang-format directly.

Reply:
I also added cppcheck